src/config.cr

```diff
@@ -18,6 +9,14 @@ require "./drivers"
 # Server required after application controllers
 require "action-controller/server"
 
+PROD = ENV["SG_ENV"]? == "production"
+
+# Configure logging
+Log.builder.bind "*", :warning, ActionController.default_backend
+Log.builder.bind "action-controller.*", :info, ActionController.default_backend
+
+filters = PROD ? ["bearer_token", "secret", "password"] : [] of String
+
 # Add handlers that should run before your application
 ActionController::Server.before(
   HTTP::ErrorHandler.new(!PROD),
```

Comment:
`HTTP::ErrorHandler` now expects production? not development?
So can remove the `!`